Index: index.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>var express = require('express');\nvar app = express();\nconst PORT = process.env.PORT || 3000;\n\napp.get('/', function(req,res){\n    res.send('hello');\n})\n\nconst run = app.listen(PORT, () => {\n    console.log(\"server is running!\");\n})
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/index.js b/index.js
--- a/index.js	(revision 59603aa7e47efd08a30552775233c8c90e2f1dde)
+++ b/index.js	(date 1615055124378)
@@ -1,11 +1,34 @@
 var express = require('express');
 var app = express();
 const PORT = process.env.PORT || 3000;
+var unirest = require("unirest");
 
 app.get('/', function(req,res){
     res.send('hello');
+    var reqa= unirest("GET", "https://uphere-space1.p.rapidapi.com/user/visible");
+
+    reqa.query({
+        "lat": "32.1433",
+        "lng": "-117.9833"
+    });
+
+    reqa.headers({
+        "x-rapidapi-key": "SIGN-UP-FOR-KEY",
+        "x-rapidapi-host": "uphere-space1.p.rapidapi.com",
+        "useQueryString": true
+    });
+
+    let response;
+    reqa.end(function (res) {
+        if (res.error) throw new Error(res.error);
+        response = resa.body;
+        console.log(response);
+    });
 })
 
 const run = app.listen(PORT, () => {
     console.log("server is running!");
-})
\ No newline at end of file
+
+})
+
+
Index: .idea/HackCU_2021_Sevens.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/HackCU_2021_Sevens.iml b/.idea/HackCU_2021_Sevens.iml
new file mode 100644
--- /dev/null	(date 1615054459622)
+++ b/.idea/HackCU_2021_Sevens.iml	(date 1615054459622)
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$">
+      <excludeFolder url="file://$MODULE_DIR$/temp" />
+      <excludeFolder url="file://$MODULE_DIR$/.tmp" />
+      <excludeFolder url="file://$MODULE_DIR$/tmp" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
new file mode 100644
--- /dev/null	(date 1615054459628)
+++ b/.idea/modules.xml	(date 1615054459628)
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/HackCU_2021_Sevens.iml" filepath="$PROJECT_DIR$/.idea/HackCU_2021_Sevens.iml" />
+    </modules>
+  </component>
+</project>
\ No newline at end of file
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
new file mode 100644
--- /dev/null	(date 1615054459632)
+++ b/.idea/vcs.xml	(date 1615054459632)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1615054459941)
+++ b/.idea/.gitignore	(date 1615054459941)
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
